### ISSUE TITLE
how to add cache tab

### ISSUE THREAD

**Issue by xralphack** [barryvdh/laravel-debugbar#719]
there is no cache config in config/debugbar.php

I can't understand where should I put the following settings

If you want you can add your own DataCollectors, through the Container or the Facade:
```
Debugbar::addCollector(new DebugBar\DataCollector\MessagesCollector('my_messages'));
//Or via the App container:
$debugbar = App::make('debugbar');
$debugbar->addCollector(new DebugBar\DataCollector\MessagesCollector('my_messages'));
```

Can someone help me?

**Comment by aik099** [comment]
There is cache setting for this: https://github.com/barryvdh/laravel-debugbar/blob/master/config/debugbar.php#L119

If you've published DebugBar config before `Cache` tab was added, then your config defaults that setting to false.

Here is PR changing the config: #678

**Comment by xralphack** [comment]
@aik099 
I didn't see anything about cache in config file.
I will try to reinstall again.
Thanks for your explanation.

**Comment by xralphack** [comment]
I installed version 2.4 again.

config still have nothing about cache.

**Comment by aik099** [comment]
@xralphack , do you see, that `cache` related setting in `config/debugbar.php` are present (see https://github.com/barryvdh/laravel-debugbar/blob/master/config/debugbar.php#L119)?

I'm suspecting that cache tab is only present in 3.0.x release that will work with Laravel 5.5+ only.

**Comment by Broutard** [comment]
You are right @aik099, cache tab is implemented since 3.0.x (and the ability to clear keys only in master branch for now)
